net-libs/libsrsirc/files/libsrsirc-0.0.14-openssl-libs-configure.patch

```diff
@@ -0,0 +1,20 @@
+--- libsrsirc-0.0.14.orig/configure	2024-04-04 20:07:02.865826089 +0000
++++ libsrsirc-0.0.14/configure	2024-04-04 20:08:24.153354192 +0000
+@@ -12216,7 +12216,7 @@
+   $as_echo_n "(cached) " >&6
+ else
+   ac_check_lib_save_LIBS=$LIBS
+-LIBS="-lssl  $LIBS"
++LIBS="-lssl -lcrypto $LIBS"
+ cat confdefs.h - <<_ACEOF >conftest.$ac_ext
+ /* end confdefs.h.  */
+ 
+@@ -12251,7 +12251,7 @@
+ #define HAVE_LIBSSL 1
+ _ACEOF
+ 
+-  LIBS="-lssl $LIBS"
++  LIBS="-lssl -lcrypto $LIBS"
```

Comment:
Its probably better to fix this in the configure.ac so that it can be upstreamed.

Reply:
There's a problem with that:
```
> grep "lssl" libsrsirc-0.0.14/configure.ac
#               LIBS="$LIBS -lssl"
```
There's exactly one occurrence of a suspicious line in configure.ac and it's commented out.
And I was unable to reproduce the the linking bug on upstream code with same CFLAGS and compiler.

Reply:
It comes from here.

https://github.com/fstd/libsrsirc/blob/e76c45dd0598cf28695eed12cb0d2e05f29cace9/configure.ac#L37

The commented code is the old check.